spec/features/navigation_spec.rb

```diff
@@ -43,4 +43,67 @@
       end
     end
   end
+
+
+  describe "as a merchant employee" do
+    it "When I click on the register link in the nav bar" do
+      merchant_1 = User.create(name: 'Bill Gates',
+                          address: '1000 Microsoft Drive',
+                          city: 'Seattle',
+                          state: 'WA',
+                          zip: '00123',
+                          email: 'bill.gates@outlook.com',
+                          password: '@%)abc123#$.',
+                          role: 1)
+
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(merchant_1)
+
+
+      visit '/items'
+
+      within 'nav' do
+        expect(page).to have_content("Merchant Dashboard")
+      end
+
+      within 'nav' do
+        click_link 'Merchant Dashboard'
+        expect(current_path).to eq('/merchant')
+      end
+    end
+  end
+
+
+  describe 'As an Admin' do
```

Comment:
This is part of user story 5. 

- a link to my admin dashboard ("/admin")
- a link to see all users ("/admin/users")

There should be a test that there is NOT the following:
- a link to my shopping cart ("/cart") or count of cart items